### PR TITLE
Cody for VSC 1.14.0 blog

### PR DESCRIPTION
**Pull request by iskyOS**
Adds the blog post for Cody for VS Code 1.14.0.

### CODE REVIEW

File: content/blogposts/2024/cody-vscode-1-14-0-release.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  src="https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1-14-release/new-chat-ui.png"
+  alt="Cody's new chat interface design"
+/>
+
+## Unit test command improvements for JS, TS, Go, and Python
+
+We’ve improved Cody’s unit test command for JavaScript, TypeScript, Go, and Python code in two ways:
+
+1. Cody will now determine the correct range of code to apply the command to based on the code syntax
+2. When you trigger a Quick Action (Ctrl + . / Cmd + .) on a function symbol, Cody will now given an option to generate a unit test
```
Comment (jdorfman) [inline content/blogposts/2024/cody-vscode-1-14-0-release.md]: Replace with `<kbd>Ctrl</kbd> + <kbd>.</kbd>`

File: content/blogposts/2024/cody-vscode-1-14-0-release.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v1.14.0 is now available! This update includes context window expansions, a new chat interface, easier ways to add context to chat, new models, and more.
+
+## Bigger and better context windows to improve chat
+
+We’ve historically limited Cody to a maximum context window of 7,000 tokens, meaning that Cody won’t pass more than 7,000 tokens (small chunks of text) to the underlying LLMs. This is mostly to make sure that _too much_ context isn’t passed into the LLM, which can lead to poor context recall and then poor answer quality. 
+
+Because of Claude 3’s [improved context recall](https://www.google.com/url?q=https://www.anthropic.com/news/claude-3-family%23:~:text%3DLong%2520context%2520and%2520near%252Dperfect%2520recall&sa=D&source=docs&ust=1713389874874139&usg=AOvVaw2xo3VN2mbBXrMP3p9cCkLc), we can now expand that context window without degrading chat quality. We’ve made huge updates to Cody’s maximum context window for Claude 3 Sonnet and Opus models:
+
+* 30,000 tokens of user-defined context (user text plus @-mentioned files)
```
Comment (jdorfman) [inline content/blogposts/2024/cody-vscode-1-14-0-release.md:25]: I usually wrap `@mention-s` in backticks

**Review by jdorfman — DISMISSED** [review]
Almost there!

File: content/blogposts/2024/cody-vscode-1-14-0-release.md (hunk 2 above)
Comment (iskyOS) [reply content/blogposts/2024/cody-vscode-1-14-0-release.md]: Resolved in new commit

File: content/blogposts/2024/cody-vscode-1-14-0-release.md (hunk 1 above)
Comment (iskyOS) [reply content/blogposts/2024/cody-vscode-1-14-0-release.md]: Resolved in new commit

**Pull request merged**